src/components/LoginHero/LoginHero.js

```diff
@@ -0,0 +1,47 @@
+import React, { useEffect, useRef } from 'react';
+import './LoginHero.scss';
+
+const d = 300
+
+function randomDist(dist) {
+    return (Math.random() * 2 - 1) * dist
+}
+
+export default function LoginHero() {
+    let messages = ["Hello Nimbus", "Welcome to CSAI", "Check us out on Github"];
+    let index = Math.floor(Math.random() * messages.length)
+    let messageRef = useRef(null)
+    let currentMessage = messages[index].split('').map((char, i) => <span className='letter' style={{
+        display: `${char === ' ' ? 'initial' : ''}`
+    }} key={i} > {char}</ span >)
+
+    let animateLetters = () => {
+        let letters = [...messageRef.current.children]
+        letters.forEach(letter => {
+            // animate each individual span element (character)
+            let keyframes = {
+                transform: [`translate(${1 * randomDist(d)}px, ${1 * randomDist(d)}px)`, 
+                `translate(${0.8911 * randomDist(d)}px, ${0.8911 * randomDist(d)}px)`,
+                `translate(${0.5644 * randomDist(d)}px, ${0.5644 * randomDist(d)}px)`,
+                `translate(${0.0299 * randomDist(d)}px, ${0.0199 * randomDist(d)}px)`,
+                `translate(${0.0066 * randomDist(d)}px, ${0.0066 * randomDist(d)}px)`,
+                `translate(${0.0163 * randomDist(d)}px, ${0.0163 * randomDist(d)}px)`,
+                `translate(${0.0154 * randomDist(d)}px, ${0.0154 * randomDist(d)}px)`,
+                `translate(0, 0)`],
```

Comment:
I agree, it would be nice if we could make that happen. We weren't able to figure out a way to get that behavior with `cubic-bezier`. Check out the [easing function](https://easings.net/#easeInBounce) we were attempting to emulate.